test/plugin/test_parser_json.rb

```diff
@@ -135,4 +135,60 @@ def test_yajl_parse_io_with_buffer_smaller_than_input
       end
     end
   end
+
+  sub_test_case "various record pattern" do
+    data("Only string", { record: '"message"', expected: [nil] }, keep: true)
+    data("Only string without quaot", { record: "message", expected: [nil] }, keep: true)
```

Comment:
`quaot`
Is it a typo for "quote"?

Reply:
Thanks! This is a typo! Now I fixed this!